clients/admin-ui/src/features/privacy-requests/RequestFilters.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  Flex,
+  Text,
+  Button,
+  Select,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  InputLeftAddon,
+  Stack,
+} from '@fidesui/react';
+
+import PIIToggle from './PIIToggle';
+import {
+  DownloadSolidIcon,
+  CloseSolidIcon,
+  SearchLineIcon,
+} from '../common/Icon';
+import { statusPropMap } from './RequestBadge';
+
+import { PrivacyRequestStatus } from './types';
+import {
+  setRequestStatus,
+  setRequestId,
+  setRequestFrom,
+  setRequestTo,
+  clearAllFilters,
+  selectPrivacyRequestFilters,
+  requestCSVDownload,
+} from './privacy-requests.slice';
+import { selectUserToken } from '../user/user.slice';
+
+const useRequestFilters = () => {
+  const filters = useSelector(selectPrivacyRequestFilters);
+  const token = useSelector(selectUserToken);
+  const dispatch = useDispatch();
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    dispatch(setRequestId(event.target.value));
+  };
+  const handleStatusChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    dispatch(setRequestStatus(event.target.value as PrivacyRequestStatus));
+  };
+  const handleFromChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Also, are we handling the case where someone might want to search by just one date field?

Reply:
Ah, I think that would probably create more complexity than it's worth, because then we'd have to add logic to distinguish between changing the `from` date change and the `to` date change. 

And yes, if a user specifies only one date field, only that date field will be included as a param. You can see that logic in the [`mapFiltersToSearchParams`](https://github.com/ethyca/fidesops/pull/364/files#diff-8304db3d5ee974fc8bf6614b0d59aea2962e5919d4bbf411cd0f8228d12eb70aR14) function.

Reply:
I just added a PR for this on the backend: https://github.com/ethyca/fidesops/pull/376. It'd be good to show an error of some kind in the UI eventually too.